README.md

```diff
@@ -5,7 +5,7 @@ If you’re an open source observability user or contributor, we invite you to g
 * Ask a question by filing an [issue](https://github.com/aws-observability/aws-otel-community/issues).
 * File a bug by filing a [pull request (PR)](https://github.com/aws-observability/aws-otel-community/pulls).
 * Contribute an enhancement or a feature which you need and maintainers will be happy to code review. You can open an issue to discuss the design of your proposed enhancement and then file a PR.
-* If you’re just getting started, we welcome you to get started with issues tagged as “good first issues” and join [Gitter](https://gitter.im/aws-otel-distro/community) to ask maintainers and other developers any questions you may have.
+* If you’re just getting started, we welcome you to get started with issues tagged as “good first issue”.
```

Comment:
Here, we should call out that customers can use the [CNCF Slack community](https://opentelemetry.io/community/#develop-and-contribute), specifically the `#opentelemetry` channel for interactive discussions.